packages/fdc3-agent-proxy/src/Messaging.ts

```diff
@@ -1,6 +1,6 @@
 import { Connectable, AppIdentifier, ImplementationMetadata } from "@kite9/fdc3-standard";
 import { RegisterableListener } from "./listeners/RegisterableListener";
-import { AddContextListenerRequest, AddContextListenerRequestMeta } from "@kite9/fdc3-schema/generated/api/BrowserTypes";
+import { AddContextListenerRequestMeta } from "@kite9/fdc3-schema/generated/api/BrowserTypes";
```

Comment:
Only top-level types should be imported from BrowserTypes (as the naming of the lower-level types can be unstable),. Hence, I'd import `AppRequestMessage` and then extract the type for its meta element:

```ts
import { AppRequestMessage} from "@kite9/fdc3-schema/generated/api/BrowserTypes";

type RequestMetadata = AppRequestMessage["meta"];
```
then use that down on line 33.